app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/AnonymizeAction.jsx

```diff
@@ -0,0 +1,56 @@
+import { useMutation } from '@tanstack/react-query';
+import React, { useState } from 'react';
+import { Button, Confirm, Message } from 'semantic-ui-react';
+import Errored from '../../../Requests/Errored';
+import Loading from '../../../Requests/Loading';
+import anonymize from './api/anonymize';
+
+export default function AnonymizeAction({ userId, wcaId }) {
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const [completed, setCompleted] = useState(false);
+  const [newWcaId, setNewWcaId] = useState();
```

Comment:
And this can be replaced with the mutation's `data.new_wca_id`.

Reply:
Oh that's nice too, changed. Thanks.